HACKING.md

```diff
@@ -41,50 +43,47 @@ To check if the watcher daemon is running, use `docker ps`. Then you can use `do
 
 ### Running the site with native python
 
-Since the site is basically a Django app, you can also run the site in the traditional way using [python 2.7](https://www.python.org/download/releases/2.7/) and [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/):
+Since the site is basically a Flask app, you can also run the site in the traditional way using [python 3](https://docs.python.org/3/) and [venv](https://docs.python.org/3/library/venv.html?highlight=venv#module-venv):
 
 ``` bash
-virtualenv env
+python3 -m venv env
 source env/bin/activate
 pip install -r requirements.txt
 ./entrypoint 0.0.0.0:8001
 ```
 
 Now browse to the site at <http://127.0.0.1:8001>. If it looks a bit odd, it's probably because you haven't built sass - see below.
```

Comment:
Is it not http://0.0.0.1:8001 ?

Reply:
well, I don't think it matters, but for consistencies sake, they should be the same.

Reply:
@nottrobin thanks for the explanations. I didn't know that. Every day is a school day :+1:

Reply:
Just to add a little more flavour - the practical use for `0.0.0.0` is for when you want to tell a server which requests to listen for. If you tell it to listen at `127.0.0.1` then it will only listen for incoming requests with `Host: 127.0.0.1` headers. If you tell it to listen at `0.0.0.0` then it will listen for incoming requests with any `Host` header.

This is why we run Talisker with `--bind 0.0.0.0:8001`, and why what gets output on the command-line is `http://0.0.0.0:8001`. And the fact that that pattern is so common in local development is why browsers tend to assume that people visiting `http://0.0.0.0` probably want the local machine.